istio-cni/templates/istio-cni.yaml

```diff
@@ -27,9 +85,6 @@ spec:
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        {{- if .Values.cni.podAnnotations }}
```

Comment:
Removing these 3 lines is a regression. It seems like you've not copied the `master` version of the file from `istio/cni`.

Reply:
And for some reason the bot failed to create a cherry-pick commit for https://github.com/istio/cni/pull/206. Just triggered it. We'll need to merge https://github.com/istio/cni/pull/215 for 1.4.0.

Reply:
done, thanks!